docs/api/dashboard-import/export.asciidoc

```diff
@@ -13,6 +13,10 @@ Note: You cannot access this endpoint via the Console in Kibana.
 
 `GET /api/kibana/dashboards/export`
```

Comment:
To make all of these more clear, I would use the following format:

When targeting the `default` {kib} space:

`GET /api/kibana/dashboards/export`

When not targeting the {kib} `default` space:

`GET /s/<space id>/api/kibana/dashboards/export`